### PR TITLE
Show progress on physical hardware

### PR DESCRIPTION
**Pull request by carlesba**

### CODE REVIEW

File: color.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+  constructor (value) {
+    this.value = value
+  }
+  transitionTo (color, ratio) {
+    var a = removeHash(this.value)
+    var b = removeHash(color.value)
+    return transitionColor(a, b, stepRatio(ratio))
+  }
+}
+
```
Comment (carlesba) [inline color.js:11]: this is copypasted… sorry

File: color.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+  return hex(r) + hex(g) + hex(b)
+}
+
+function hex (x) {
+  x = x.toString(16);
+  return (x.length == 1) ? '0' + x : x;
+}
+
+function removeHash (colorWithHash) {
```
Comment (davesnx) [inline color.js:25]: this shoudn't be part of the color interface?

File: color.js [hunk 3]
```diff
@@ -0,0 +1,35 @@
+class Color {
+  constructor (value) {
+    this.value = value
+  }
+  transitionTo (color, ratio) {
+    var a = removeHash(this.value)
```
Comment (epallerols) [inline color.js:6]: 🍅

**Comment by epallerols** [comment]
🍅

**Pull request merged**